black.py

```diff
@@ -1028,11 +1033,11 @@ class BracketTracker:
     """Keeps track of brackets on a line."""
 
     depth: int = 0
-    bracket_match: Dict[Tuple[Depth, NodeType], Leaf] = Factory(dict)
-    delimiters: Dict[LeafID, Priority] = Factory(dict)
+    bracket_match: Dict[Tuple[Depth, NodeType], Leaf] = field(default_factory=dict)
+    delimiters: Dict[LeafID, Priority] = field(default_factory=dict)
     previous: Optional[Leaf] = None
-    _for_loop_depths: List[int] = Factory(list)
-    _lambda_argument_depths: List[int] = Factory(list)
+    _for_loop_depths: List[int] = field(default_factory=list)
+    _lambda_argument_depths: List[int] = field(default_factory=list)
```

Comment:
_Maybe_ it makes sense to define a helper:
```python
def Factory(fn):
    return field(default_factory=fn)
```
to minimize the size of the diff? Not sure however what looks better.

Reply:
I think this code is fine. I'd rather use dataclasses idiomatically than minimize diff size.